src/OrchardCore/OrchardCore.ContentManagement.Display/Razor/OrchardRazorHelperExtensions.cs

```diff
@@ -0,0 +1,20 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Html;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.ContentManagement.Display;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Razor;
+
+namespace OrchardCore.ContentManagement
+{
+    public static class OrchardRazorHelperExtensions
+    {
+        public static async Task<IHtmlContent> DisplayAsync(this OrchardRazorHelper razorHelper, ContentItem content, string displayType = "", string groupId = "", IUpdateModel updater = null)
```

Comment:
If someone can think of a good way to automatically assign the current `IUpdateModel` (if there is one) to `updater`, I'll update this method to do so.

Reply:
I asked me the same for the related liquid filter `BuildDisplayFilter` but i thought it was ok to pass null because we are not doing a `BuildEditorAsync()`. Otherwise, never tried but there is an `IUpdateModelAccessor` service, see in `DisplayContentItemViewComponent` which uses it.